infrastructure/data_access/http/synthetics_repo.py

```diff
@@ -112,17 +144,16 @@ def scale_to_percents(val: str) -> MetricValue:
     return MetricValue(float(val) * 100.0)
 
 
-def make_internal_agents(agents, agent_ids) -> Agents:
+def transform_to_internal_agents(agents) -> Agents:
     result = Agents()
     for agent in agents:
-        if agent.id in agent_ids:
-            result.insert(
-                Agent(
-                    id=AgentID(agent.id),
-                    ip=agent.ip,
-                    name=agent.name,
-                    alias=agent.alias,
-                    coords=Coordinates(agent.long, agent.lat),
-                )
+        result.insert(
```

Comment:
It is somewhat wasteful to create an entry for every agent in the Kentik account only the filter most of them out almost immediately. There are many more agents available in the Kentik system than used in any single mesh test. The previous approach was IMHO better.